### PR TITLE
Unused methods dataset page cleanup.

### PR DESCRIPTION
**Pull request by MrK191**
- I was able to create and view dataset without a problem, so there shouldn't be any suprises.

### CODE REVIEW

File: dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java [hunk 1]
```diff
@@ -3479,48 +2701,6 @@ public boolean isTwoRavenAmongExternalTools(List<ExternalTool> externalTools) {
...
-            // whatever...
-        }
-
-        thisLatestReleasedVersion = workingVersion.equals(latestPublishedVersion);
-
-        return thisLatestReleasedVersion;
-
-    }
-
-    public String getJsonLd() {
```
Comment (madryk) [inline dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java:3171]: I've got
```
        <ui:define name="jsonld_header">
            <script type="application/ld+json">
                <h:outputText value="#{DatasetPage.jsonLd}"/>
            </script>
        </ui:define>
```
in dataset.xhtml - looks like it is used

File: dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java [hunk 2]
```diff
@@ -2507,171 +2067,23 @@ public Boolean isDatasetPublishPopupCustomTextOnAllVersions() {
...
-        // And reset the selected fileMetadata:
-
-        fileMetadataSelectedForThumbnailPopup = null;
-    }
 
     /*
      * Items for the "Tags (Categories)" popup.
      *
      */
     private FileMetadata fileMetadataSelectedForTagsPopup = null;
```
Comment (madryk) [inline dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java]: Probably we can get rid of this field too - it's always null

File: dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java [hunk 3]
```diff
@@ -2507,171 +2067,23 @@ public Boolean isDatasetPublishPopupCustomTextOnAllVersions() {
...
-            }
-        }
-        return false;
-    }
-
     /*
      * Items for the "Designated this image as the Dataset thumbnail:
      */
 
     private FileMetadata fileMetadataSelectedForThumbnailPopup = null;
```
Comment (madryk) [inline dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java]: Probably we can get rid of this field too - it's always null

File: dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java [hunk 4]
```diff
@@ -1015,13 +719,13 @@ public String initCitation() {
 
     private String init(boolean initFull) {
         //System.out.println("_YE_OLDE_QUERY_COUNTER_");  // for debug purposes
-        this.maxFileUploadSizeInBytes = settingsService.getValueForKeyAsLong(SettingsServiceBean.Key.MaxFileUploadSizeInBytes);
+        Long maxFileUploadSizeInBytes = settingsService.getValueForKeyAsLong(SettingsServiceBean.Key.MaxFileUploadSizeInBytes);
```
Comment (madryk) [inline dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java]: looks like this is not used

File: dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java [hunk 5]
```diff
@@ -373,16 +295,6 @@ public void setRemoveUnusedTags(boolean removeUnusedTags) {
     private String fileSortField;
     private String fileSortOrder;
```
Comment (madryk) [inline dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java]: These two fields are populated via query string but are in no way interpreted - can be removed

**Review by madryk — CHANGES_REQUESTED** [review]

File: dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java (hunk 1 above)
Comment (MrK191) [reply dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DatasetPage.java:3171]: Yes I noticed when I tried to create dataset, I re-added it in 2nd commit, it is just in different place.

**Review by madryk — APPROVED** [review]

**Pull request merged**